packages/docs/src/pages/tutorial/hooks/use-document/problem/app.tsx

```diff
@@ -0,0 +1,9 @@
+import { component$, useHostElement, Host } from '@builder.io/qwik';
+
+export const App = component$(
+  () => {
+    const host = useHostElement();
+    return <Host>Host element tag-name: {host.tagName}</Host>;
+  },
+  { tagName: 'my-app' }
+);
```

Comment:
There seem to be a copy-paste issue as the solution and the problem are the same

Reply:
Thanks for the suggestion. I have incorporated it into the PR.